spec/features/users/user_edit_spec.rb

```diff
@@ -1,20 +1,17 @@
 include Warden::Test::Helpers
 Warden.test_mode!
 
-# Feature: User edit
-#   As a user
-#   I want to edit my user profile
-#   So I can change my email address
-feature 'User edit', :devise do
-
+feature 'User edit', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.